crates/runtime/src/lib.rs

```diff
@@ -379,6 +382,11 @@ impl Runtime {
         Arc::clone(&self.app)
     }
 
+    #[must_use]
+    pub fn accelerator_engine_registry(&self) -> AcceleratorEngineRegistry {
```

Comment:
```suggestion
    pub(crate) fn accelerator_engine_registry(&self) -> AcceleratorEngineRegistry {
```

I don't think we need to expose this? Could we make this only visible to tests as well (I'm not sure what else requires this?)

Reply:
It's needed in spicecloud crate when using `create_synced_internal_accelerated_table`